docs/project/repository-guide.md

```diff
@@ -0,0 +1,77 @@
+
+- [Fetching or contributing to IPFS code](#fetch-or-contribute-to-ipfs-code)
+- [Participating in IPFS project operations](#participate-in-ipfs-project-operations)
+- [Discussing ideas for using IPFS](#discuss-ideas-for-using-ipfs)
+
+## Fetch or contribute to IPFS code
+
+Organization and repository links for the top-level projects shipped as part of the IPFS project and the main repos for each ingredient in the stack upon which IPFS is built.
+
+### Protocol implementations
+
+- [go-ipfs](https://github.com/ipfs/go-ipfs): The reference implementation written in Go.
+- [js-ipfs](https://github.com/ipfs/js-ipfs): The JavaScript implementation of IPFS.
+- [rust-ipfs](https://github.com/rs-ipfs/rust-ipfs): Alpha implementation in Rust.
+
+### Client implementations
+
+- [Current list](https://github.com/ipfs/ipfs#http-client-libraries) of HTTP client libraries.
+
+### Underlying components
+
+- [libp2p](https://github.com/libp2p):
+    - [go-libp2p](https://github.com/libp2p/go-libp2p): Reference libp2p implementation in Go.
+    - [js-libp2p](https://github.com/libp2p/js-libp2p): The JavaScript implementation of the libp2p networking stack.
+    - [rust-libp2p](https://github.com/libp2p/rust-libp2p): The Rust implementation of the libp2p networking stack.
+- [IPLD](https://github.com/ipld):
+    - [go-ipld](https://github.com/ipld/go-ipld): Entry-point repo for Go IPLD development.
+    - [js-ipld](https://github.com/ipld/js-ipld): The JavaScript Implementation of IPLD.
+    - [IPLD specifications](https://github.com/ipld/specs): The set of specifications that make up IPLD.
+- [Multiformats](https://github.com/multiformats):
+    - [Multiaddr](https://github.com/multiformats/multiaddr): Composable and future-proof network addresses.
+    - [Multibase](https://github.com/multiformats/multibase): Self-identifying base encodings.
+    - [Multicodec](https://github.com/multiformats/multicodec): Compact, self-describing codecs.
+    - [Multihash](https://github.com/multiformats/multihash): Self-describing hashes for future-proofing.
+
+### IPFS tools and products
+
+- [IPFS Cluster](https://github.com/ipfs/ipfs-cluster): Automatically allocate, replicate, and track your data as a global pinset distributed among a swarm of peers.
+- [IPFS Companion](https://github.com/ipfs/ipfs-companion): The IPFS web browser extension.
+- [IPFS Web UI](https://github.com/ipfs/ipfs-webui): An easy-to-use web interface for IPFS nodes.
+- [IPFS Desktop](https://github.com/ipfs/ipfs-desktop): Standalone IPFS app with an easy-to-use node interface, plus menubar/tray shortcuts
+- [ipfs-gui](https://github.com/ipfs/ipfs-gui): Coordinating development, user experience, and maintenance of IPFS GUIs.
+- [i18n](https://github.com/ipfs-shipyard/i18n): The IPFS Translation Project, crowdsourcing translations of IPFS GUIs and websites.
```

Comment:
i18n should probably be pulled out into a separate subsection, maybe the Participate section below?